job/server/src/main/java/alluxio/worker/job/RetryHandlingJobMasterClient.java

```diff
@@ -36,7 +36,7 @@
  * workers.
  */
 @ThreadSafe
-public final class RetryHandlingJobMasterClient extends AbstractMasterClient
+public final class RetryHandlingJobMasterClient extends AbstractJobMasterClient
```

Comment:
There is another job-master client with the same name under:
`src/main/java/alluxio/client/job/RetryHandlingJobMasterClient.java`

Reply:
Very good finding, that explains why some tests failed.